File: src/main/webapp/js/service/AttachmentService.js
//服务层
app.service('attachmentService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../attachment/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../attachment/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../attachment/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../attachment/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../attachment/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../attachment/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../attachment/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});

File: src/main/java/com/wyan/service/AttachmentService.java
package com.wyan.service;

import com.github.pagehelper.PageInfo;
import com.wyan.pojo.TbAttachment;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface AttachmentService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbAttachment> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbAttachment attachment);


	/**
	 * 修改
	 */
	public void update(TbAttachment attachment);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbAttachment findOne(Integer aid);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] aids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbAttachment attachment, int pageNum, int pageSize);
	
}
